Makefile

```diff
@@ -79,6 +79,11 @@ pmix-standard.pdf: $(CHAPTERS) $(SOURCES) pmix.sty pmix-standard.tex figs/pmix-l
 	$(LATEX_C) -interaction=batchmode pmix-standard.tex
 	@cp pmix-standard.pdf pmix-standard-${version}.pdf
 
+release:
+	@./bin/release.sh set
+	@make all
+	@./bin/release.sh restore
```

Comment:
I'm going to make a small change to this create the release in a temporary directory instead of the working copy. This way if something goes wrong, it won't pollute the state of your repo. This is the way the MPICH release script works.